doc/source/user_guide/enhancingperf.rst

```diff
@@ -562,6 +562,59 @@ Now let's do the same thing but with comparisons:
    of type ``bool`` or ``np.bool_``. Again, you should perform these kinds of
    operations in plain Python.
 
+The ``Series.eval`` method
```

Comment:
i would not add any of this, you can simply remove the reference to DataFrame below (call it query).

Reply:
Jus to clarify, which 'reference to DataFrame' did you mean to remove?